development/print_db_data.py

```diff
@@ -122,50 +122,36 @@ def print_production_lines():
 def print_verbose_lines():
 	print('Data has been moved, this view is unavailable for now')
 	return
-	def _output_helper_prodlines(string, list):
```

Comment:
When a function calls return , processing for that function stops and control is returned to the code that called the function. Any code that is placed after a return statement will never be executed. If any part of the program could potentially rely on this code, then the code must be moved to a location where it can be reached. If the code is unnecessary, then it should be removed, because extra code reduces readability.